.travis.yml

```diff
@@ -36,7 +36,6 @@ script:
   - "sh -e /etc/init.d/xvfb start"
   - sleep 3 # give xvfb some time to start
   - nvm install $NODE_VERSION
-    && ci/change_version.sh -m . -swarm lib/taskana-cdi/src/test/java/pro/taskana/TaskanaProducersTest.java
```

Comment:
Only the swarm parameter please. We still need to change the version when we have a release build

Reply:
We also have 3 occurances of the  -swarm parameter. You just removed one of them. :)